package.json

```diff
@@ -40,6 +40,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^3.0.0",
     "ganache-cli": "^6.12.2",
+    "hardhat": "^2.0.8",
```

Comment:
I installed `hardhat` to try to fix the Slither issue, but it didn't work. This dependency might not be necessary after all.

Reply:
I could see it being useful to have it be a top-level dev dependency and/or being a devDependency of every package where it isn't needed as a runtime dep.